src/cashier/currency.es6

```diff
@@ -4,6 +4,7 @@ import windows from 'windows/windows';
 import rv from 'common/rivetsExtra';
 import html from 'text!cashier/currency.html';
 
+require(['common/util'])
 require(['text!cashier/currency.html']);
```

Comment:
@4p00rv Why not we delete that require() statement and keep that import and see if it works. I would prefer to use the import rather than require to reduce one network call here